src/components/TeacherComponents/NewContentBanner/NewContentBanner.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { FC } from "react";
+import styled from "styled-components";
+import {
+  OakFlex,
+  OakHeading,
+  OakP,
+  OakTertiaryButton,
+  OakImage,
+} from "@oaknational/oak-components";
+
+import { resolveOakHref, OakPageType } from "@/common-lib/urls";
+
+const StyledOakFlex = styled(OakFlex)`
+  box-shadow: 0px 0px 8px 0px rgba(0, 0, 0, 0.2);
+`;
+
+type NewContentBannerProps = {
+  subjectTitle: string;
+  programmeSlug: string;
+  keyStageSlug: string;
+  subjectSlug: string;
+  isUnitListing?: boolean;
+};
+
+const renderContentBannerArray = [
+  { keyStageSlug: "ks1", subjectSlug: "english" },
+  { keyStageSlug: "ks1", subjectSlug: "geography" },
+  { keyStageSlug: "ks1", subjectSlug: "history" },
+  { keyStageSlug: "ks1", subjectSlug: "science" },
+  { keyStageSlug: "ks2", subjectSlug: "english" },
+  { keyStageSlug: "ks2", subjectSlug: "english-grammar" },
+  { keyStageSlug: "ks2", subjectSlug: "english-reading-for-pleasure" },
+  { keyStageSlug: "ks2", subjectSlug: "english-spelling" },
+  { keyStageSlug: "ks2", subjectSlug: "geography" },
+  { keyStageSlug: "ks2", subjectSlug: "history" },
+  { keyStageSlug: "ks2", subjectSlug: "science" },
+  { keyStageSlug: "ks3", subjectSlug: "english" },
+  { keyStageSlug: "ks3", subjectSlug: "history" },
+  { keyStageSlug: "ks3", subjectSlug: "science" },
+  { keyStageSlug: "ks4", subjectSlug: "biology" },
+  { keyStageSlug: "ks4", subjectSlug: "chemistry" },
+  { keyStageSlug: "ks4", subjectSlug: "combined-science" },
+  { keyStageSlug: "ks1", subjectSlug: "maths" },
+  { keyStageSlug: "ks2", subjectSlug: "maths" },
+  { keyStageSlug: "ks3", subjectSlug: "maths" },
+  { keyStageSlug: "ks4", subjectSlug: "maths" },
+];
```

Comment:
don't need to change since this is a temporary component (I think?) but just pointing out this seems like kind of a strange data structure to use for this! I would have thought a record would be make looking up subjects by keystage straightforward

Reply:
Changed this to a record ✅